src/javascript/app/pages/cashier/account_transfer.js

```diff
@@ -145,11 +147,61 @@ const AccountTransfer = (() => {
         el_transfer_fee.setVisibility(0);
         el_success_form.setVisibility(1);
         getElementById('transfer_info').setVisibility(0);
+        getElementById('acc_transfer_header_icon').classList.add('success');
+        getElementById('acc_transfer_header_text').textContent = localize('Successfully transferred');
+    };
+
+    const onAmountInput = (e) => {
+        const input_val = parseFloat(e.target.value);
+        calculateAmount(input_val);
+    };
+
+    const calculateAmount = (value) => {
+        const el_amount    = getElementById('amount');
+        const el_amount_to = getElementById('amount_to');
+        const exchange_rate = getPropertyValue(exchange_rates, curr_account_to);
+        const transferred_amount = (value * exchange_rate).toFixed(8);
+        if (transferred_amount && !isNaN(transferred_amount)) {
```

Comment:
I think we can just do `!isNaN(transferred_amount)`, `!isNaN(undefined)` returns `false`, so should be fine?

Reply:
and if it's undefined we have bigger issues, because on the line above `undefined.toFixed(8)` will definitely throw an error